backend/remote-state/azure/arm_client.go

```diff
@@ -109,6 +121,8 @@ func buildArmClient(ctx context.Context, config BackendConfig) (*ArmClient, erro
 }
 
 func buildArmEnvironment(config BackendConfig) (*azure.Environment, error) {
+	// TODO: can we remove this?
+	// https://github.com/hashicorp/terraform/issues/27156
```

Comment:
outside of this PR since this is already unused, but we should dig into this separately